qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -511,3 +520,26 @@ def test_client_metrics_and_metadata(self):
             if not (client_metadata[i]['valid_metrics']):
                 raise RuntimeError("valid_metrics not found!")
 
+    def test_non_existing_mds_rank(self):
+        # validate
+        valid, metrics = self._get_metrics(
+            self.verify_mds_metrics(client_count=TestMDSMetrics.CLIENTS_REQUIRED), 30)
+        log.debug("metrics={0}".format(metrics))
+        self.assertTrue(valid)
+
+        filtered_mds = '15'
+        mds_metrics = metrics['metrics']
+        if f"mds.{filtered_mds}" in list(mds_metrics):
+            raise RuntimeError(f"filtered_mds={filtered_mds} already exists")
```

Comment:
I think the above test could be avoided, at-least in this test function. Because you are using `active_mds_count=1` here, so I don't think it would be a rank `15`!. Just checking the timeout/incoming-data as mentioned in my other comment is fine.